addon/components/file-dropzone/component.js

```diff
@@ -118,16 +118,30 @@ export default Ember.Component.extend({
   didDrop({ originalEvent: evt }) {
     // Testing support for dragging and dropping images
     // from other browser windows
-    let url = evt.dataTransfer.getData('text/uri-list');
-    let html = evt.dataTransfer.getData('text/html');
-    if (html) {
+    let supportsUrls, supportsHtml, html, url;
+
+    try {
+      html = evt.dataTransfer.getData('text/html');
+      supportsHtml = true;
```

Comment:
@tim-evans In hindsight, I guess it doesn't even really make much sense to store values like `supportsHtml` and `supportsUrls` the way that I'm doing it here. The `if` tests below would still receive `undefined` `html` and `url` values.

I know in #21, you had mentioned storing `supportsHtml` and `supportsUrls` properties on the component (e.g., `this.set('supportsHtml, true)`), but I wasn't sure how you'd want to use those in context of the component.